pyproject.toml

```diff
@@ -76,28 +76,28 @@ test = "invoke tests"
 
 [tool.pdm.dev-dependencies]
 dev = [
-    "BeautifulSoup4<5.0,>=4.9",
-    "jinja2~=3.1.2",
-    "lxml<5.0,>=4.3",
-    "markdown~=3.4.3",
-    "typogrify<3.0,>=2.0",
-    "sphinx<6.0,>=5.1",
-    "furo==2023.03.27",
-    "livereload<3.0,>=2.6",
-    "psutil<6.0,>=5.7",
-    "pygments~=2.15",
-    "pytest<8.0,>=7.1",
-    "pytest-cov<5.0,>=4.0",
-    "pytest-sugar<1.0.0,>=0.9.5",
-    "pytest-xdist<3.0,>=2.0",
-    "tox<4.0,>=3.13",
-    "flake8<4.0,>=3.8",
-    "flake8-import-order<1.0.0,>=0.18.1",
-    "invoke<3.0,>=2.0",
-    "isort<6.0,>=5.2",
-    "black<20.0,>=19.10b0",
-    "ruff>=0.1.3,<1.0.0",
-    "tomli;python_version<'3.11'",
+    "BeautifulSoup4>=4.12.2",
+    "jinja2>=3.1.2",
+    "lxml>=4.9.3",
+    "markdown>=3.5",
+    "typogrify>=2.0.7",
+    "sphinx>=7.1.2",
+    "furo>=2023.9.10",
+    "livereload>=2.6.3",
+    "psutil>=5.9.6",
+    "pygments>=2.16.1",
+    "pytest>=7.4.3",
+    "pytest-cov>=4.1.0",
+    "pytest-sugar>=0.9.7",
+    "pytest-xdist>=3.3.1",
+    "tox>=4.11.3",
+    "flake8>=6.1.0",
+    "flake8-import-order>=0.18.2",
+    "invoke>=2.2.0",
+    "isort>=5.12.0",
```

Comment:
Shouldn't `isort` be removed then?
```suggestion
```

Reply:
I believe so, and for that matter I think Flake8 and Black should also be removed, right @offbyone?

Reply:
I am not @offbyone, but I don't see black or isort in .pre-commit-config.yaml, so I agree.

Reply:
Correct; I left them in on the chance that someone ran them by hand, but yeah, they should be yeeted.